overlay/common/frameworks/base/core/res/res/values/config.xml

```diff
@@ -37,5 +37,19 @@
 
     <!-- Specifies the path that is used by AdaptiveIconDrawable class to crop launcher icons. -->
     <string name="config_icon_mask" translatable="false">"M50 0A50 50,0,1,1,50 100A50 50,0,1,1,50 0"</string>
+	
+	<!-- Enable doze mode
+         ComponentName of a dream to show whenever the system would otherwise have gone to sleep. -->
+    <string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
```

Comment:
update to:
`<string name="config_dozeComponent" translatable="false">com.android.systemui/com.android.systemui.doze.DozeService</string>`
also change comment to:
https://github.com/RiseOS/platform_frameworks_base/blob/2818f8d661dc6d9f0f4faedab87aa81c88b667d4/core/res/res/values/config.xml#L2089-L2097